### ISSUE TITLE
Suggestion - add option to fail when updates are found

### ISSUE THREAD

**Issue by patros** [jerriep/dotnet-outdated#93]
Hi, great tool. Thanks

I'd like to use it in some of my build configurations. What I'd really like is to fail a build if there are any update candidates found. The easiest way to do this would be to add a command line option to return a non-zero exit code when any updates are found.

I've had a look and it looks like it should be a pretty straight forward change. And it should help in any automation setting. Maybe something like these?

```csharp
[Option(CommandOptionType.NoValue, Description = "Specifies whether it should return a non-zero exit code when updates are found.", ShortName = "f", LongName = "fail-on-updates")]
public bool FailOnUpdates { get; set; } = false;
```

```csharp
private bool UpdatesExist(List<Project> projects)
{
    var dependenciesWithUpdates = projects
        .SelectMany(p => p.TargetFrameworks)
        .SelectMany(f => f.Dependencies)
        .Where(d => d.LatestVersion > d.ResolvedVersion);
    return dependenciesWithUpdates.Any();
}
```

And this at the end of OnExecute...
```csharp
if (FailOnUpdates && UpdatesExist(projects))
{
    // 1 has been used already for CommandValidationException
    return 2;
}

return 0;
```

I've made some changes locally but wanted to see what you thought before I go implement tests, pull request, etc.

**Comment by patros** [comment]
I haven't actually built and tested the above yet. Wanted to get your opinion before really doing it.

**Comment by jerriep** [comment]
Hi @patros. Yeah sure, you can submit a PR for this, thanks!

**Comment by jerriep** [comment]
Closed via #94

**Comment by jerriep** [comment]
@patros This is now available in V2.1.0 on NuGet
